src/main/java/org/mqttbee/api/mqtt/mqtt5/Mqtt5ClientBuilder.java

```diff
@@ -65,26 +68,89 @@ public Mqtt5ClientBuilder(
     }
 
     @NotNull
-    public Mqtt5ClientBuilder followingRedirects(final boolean followsRedirects) {
+    @Override
+    public Mqtt5ClientBuilder identifier(@NotNull final String identifier) {
+        super.identifier(identifier);
+        return this;
+    }
+
+    @NotNull
+    @Override
+    public Mqtt5ClientBuilder identifier(@NotNull final MqttClientIdentifier identifier) {
+        super.identifier(identifier);
+        return this;
+    }
+
+    @NotNull
+    @Override
+    public Mqtt5ClientBuilder serverHost(@NotNull final String host) {
+        super.serverHost(host);
+        return this;
+    }
+
+    @NotNull
+    @Override
+    public Mqtt5ClientBuilder serverPort(final int port) {
+        super.serverPort(port);
+        return this;
+    }
+
+    @NotNull
+    @Override
+    public Mqtt5ClientBuilder useSsl() {
+        super.useSsl();
+        return this;
+    }
+
+    @NotNull
+    @Override
+    public Mqtt5ClientBuilder useSsl(@NotNull final MqttClientSslConfig sslConfig) {
+        super.useSsl(sslConfig);
+        return this;
+    }
+
+    @NotNull
+    @Override
+    public Mqtt5ClientBuilder useWebSockets() {
+        super.useWebSockets();
+        return this;
+    }
+
+    @NotNull
+    @Override
+    public Mqtt5ClientBuilder useWebSockets(@NotNull final MqttWebsocketConfig websocketConfig) {
+        super.useWebSockets(websocketConfig);
+        return this;
+    }
+
+    @NotNull
+    @Override
+    public Mqtt5ClientBuilder executorConfig(@NotNull final MqttClientExecutorConfig executorConfig) {
+        super.executorConfig(executorConfig);
+        return this;
+    }
+
+    @NotNull
+    public Mqtt5ClientBuilder followsRedirects(final boolean followsRedirects) {
```

Comment:
If we stay with "follows" and "allows" then we should also use "usesSsl/Websocket"
or if we stay with "use" then we should also use "follow" and "allow"

Reply:
DONE